### PR TITLE
Standardize UG

### PR DESCRIPTION
**Pull request by xCelestialPhoenix**
Updated UG according to as we discussed. Added Wah Ken's portion in also which requires some confirmation.

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -16,345 +16,413 @@ endif::[]
...
 === Common
 NOVA offers a set of common functionalities across all modes.
 
 ==== Exit : `exit`
-When exiting NOVA with the `exit` command, the contacts, schedules and notes will be saved.
+You can exit NOVA with this command. While exiting NOVA, contacts, schedules and
+notes will be saved.
 
-=== Address Book
+==== Change mode of operation: `nav`
```
Comment (BryanYap972) [inline docs/UserGuide.adoc]: Can use Navigation as title instead. Will be clearer.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -16,345 +16,413 @@ endif::[]
...
+====
+* `[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
+* `[classmate/teammate]` is not case-sensitive. (E.g. "ClassMate" is the same as "classmate")
+* The name you add will be automatically formatted. (E.g. "jane doe" will become "Jane Doe")
+====
 
-* `[name]` is not case-sensitive (Jane doe is the same as jane Doe).
-* The name you add will be automatically formatted (jane doe will become Jane Doe).
+Example: +
+`contact add n\John Doe p\12345678 c\classmate`
```
Comment (lohszeying) [inline docs/UserGuide.adoc]: Remove all "contact"

**Review by BryanYap972 — COMMENTED** [review]
Rest looks ok

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -16,345 +16,413 @@ endif::[]
...
+`meeting d\CS2103T website set-up v\COM1 t\2020-02-20 14:00 1`
 
-* `meeting d\CS2103T website set-up v\COM1 t\2020-02-20 14:00 1`
+Creates an event for a team meeting at COM1 on 20 Feb 2020, 2pm, to set up CS2103T website, which last for 1 hour.
 
 ==== Add a Study Session: `study`
-
 You can add a study session as one of your events.
 
 Format: `study d\[description] v\[venue] t\[YYYY-MM-DD] [HH:MM] [duration]`
```
Comment (huixianc) [inline docs/UserGuide.adoc]: change duration to end time in HH:MM

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -16,345 +16,413 @@ endif::[]
...
-* `[index]` must be a positive integer e.g. 1, 2, 3...
+Example: +
+`done t\2020-02-20 i\2`
 
+Marks the second event on 20 Feb 2020 as completed.
 
 ==== Find Event: `find`
 You can find the events that contain the keywords.
 
 Format: `find event [keywords]`
```
Comment (huixianc) [inline docs/UserGuide.adoc]: add tag k\

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -16,345 +16,413 @@ endif::[]
...
-* `[index]` must be a positive integer e.g. 1, 2, 3...
+Example: +
+`note d\Remember to bring your charger! t\2020-02-20 i\2`
 
+Adds a note with description "Remember to bring your charger!" to the second event on 20 Feb 2020.
 
 ==== Repeat an Event: `repeat`
 You can add repeated events which occur weekly for a given number of times.
 
 Format: `repeat [number] t\[YYYY-MM-DD] i\[index]`
```
Comment (huixianc) [inline docs/UserGuide.adoc]: move number to the back and add tag \n

**Pull request merged**